repositories.yaml

```diff
@@ -1083,3 +1083,26 @@ applications:
       - v1_name: rally-attention-stream
         app_id: rally-attention-stream
     skip_documentation: true
+
+  - app_name: viu_politica
+    canonical_app_name: Viu Politica
+    app_description: |
+      This is a "spin-off" of the Mozilla YouTube RegretsReporter extension.
+      It is an opt-in browser extension where users can anonymously tag videos
+      related to the 2022 Brazilian Elections.
+    url: https://github.com/evoluchico/viu-politica
+    notification_emails:
+      - chico@mozillafoundation.org
```

Comment:
@evoluchico, do you have a team alias in MoFo we could add here as well in case there's problems with the collection and you're not around?

Reply:
We don't have a team alias in MoFo in case there's problems with the collection. But you can add Ranadheer Malla (ranadheer@mozillafoundation.org), as he's also in this project. Does that work?

Reply:
This works, thanks!